nano/node/websocket.cpp

```diff
@@ -13,6 +13,11 @@ ws_listener (listener_a), ws (std::move (socket_a)), write_strand (ws.get_execut
 
 nano::websocket::session::~session ()
 {
+	for (auto & subscription : subscriptions)
```

Comment:
This loop is missing a lock on subscriptions_mutex.

Reply:
Thanks, fixed!